Makefile

```diff
@@ -1,9 +1,20 @@
-# For generating .wasm files for parsers
-# See https://www.npmjs.com/package/web-tree-sitter
-languages = agda c clojure cpp c-sharp bash go html java javascript json markdown php python ruby rust scala sparql tsx typescript yaml
+LANGUAGES = agda c clojure cpp c-sharp bash go html java javascript json markdown php python ruby rust scala sparql tsx typescript yaml
 
-.PHONY: parsers
-parsers: $(addprefix parsers/tree-sitter-,$(addsuffix .wasm,$(languages)))
+TREE_SITTER_VERSION := v0.20.4
+
+WEB_TREE_SITTER_FILES := README.md package.json tree-sitter-web.d.ts tree-sitter.js tree-sitter.wasm
+
+
+# Build web-tree-sitter and web-tree-sitter parsers for $(LANGUAGES)
+
+.PHONY: compile
+compile: \
+		$(addprefix vendor/web-tree-sitter/,$(WEB_TREE_SITTER_FILES)) \
+		$(addprefix parsers/tree-sitter-,$(addsuffix .wasm,$(LANGUAGES)))
+	tsc -p ./
```

Comment:
why do you bake this one into the makefile now?  won't this change now cause us not to compile when we change any of our own source files, as they're not dependencies?

Reply:
Actually, `compile` is a phony task, so it’ll always run.